params/config.go

```diff
@@ -769,6 +769,10 @@ func (c *ChainConfig) IsCel2(time uint64) bool {
 	return isTimestampForked(c.Cel2Time, time)
 }
 
+func (c *ChainConfig) IsPreCel2(time uint64) bool {
+	return c.Cel2Time != nil && !c.IsCel2(time)
```

Comment:
We have a config.Celo now due to the baseFeeFloor, if we want to mimic the same logic that the optimism's team is doing.